src/compas/geometry/surfaces/__init__.py

```diff
@@ -0,0 +1,8 @@
+from ._surface import Surface  # noqa F401
+
+try:
+    import compas_occ  # noqa F401
```

Comment:
OCC is LGPL, right? I am not entirely sure this is still ok to reference explicitly without violating licenses. I guess it is, because there's a fallback to non-occ implementation... 🤷‍♀️

Reply:
https://opensource.stackexchange.com/questions/5175/including-untouched-lgpl-library-in-a-mit-licenced-project